core/extension/gdextension_interface.cpp

```diff
@@ -1036,6 +1125,10 @@ static void gdextension_ref_set_object(GDExtensionRefPtr p_ref, GDExtensionObjec
 	ref->reference_ptr(o);
 }
 
+static GDExtensionObjectPtr gdextension_callable_custom_create(GDExtensionCallableCustomInfo *p_custom_callable_info) {
```

Comment:
This wont work, return type should be `GDExtensionTypePtr`, as `Callable`s are `Variant` values not `Object` instances. However you can't return a `GDExtensionTypePtr` directly (who would own the memory?), one would need to be passed in, and then constructed into.